protx-client/src/components/ProTx/components/maps/MapProviders.jsx

```diff
@@ -4,60 +4,31 @@ export default function Maptiles() {
   const providers = [];
   const layers = {};
 
-  // Provider: Stamen.
+  // Provider: Open Street Maps
   // No key required.
-  // eslint-disable-next-line no-unused-vars
-  const basemapToner = L.tileLayer(
-    'https://stamen-tiles-{s}.a.ssl.fastly.net/toner/{z}/{x}/{y}{r}.{ext}',
-    {
-      attribution:
-        'Map tiles by <a href="http://stamen.com">Stamen Design</a>, <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
-      subdomains: 'abcd',
-      minZoom: 0,
-      maxZoom: 20,
-      ext: 'png',
-      name: 'Stamen - Toner',
-    }
-  );
-
-  const basemapTonerLite = L.tileLayer(
-    'https://stamen-tiles-{s}.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}{r}.{ext}',
-    {
-      attribution:
-        'Map tiles by <a href="http://stamen.com">Stamen Design</a>, <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
-      subdomains: 'abcd',
-      minZoom: 0,
-      maxZoom: 20,
-      ext: 'png',
-      name: 'Stamen - Toner Light',
-    }
-  );
 
   // eslint-disable-next-line no-unused-vars
-  const basemapTerrain = L.tileLayer(
-    'https://stamen-tiles-{s}.a.ssl.fastly.net/terrain/{z}/{x}/{y}{r}.{ext}',
+  const basemapOsmDefault = L.tileLayer(
+    'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
     {
       attribution:
-        'Map tiles by <a href="http://stamen.com">Stamen Design</a>, <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
-      subdomains: 'abcd',
+        'Map data: &copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
       minZoom: 0,
-      maxZoom: 18,
-      ext: 'png',
-      name: 'Stamen - Terrain',
+      maxZoom: 17,
+      name: 'OSM - Default',
+      type: 'png',
     }
   );
 
-  // Provider: Open Street Maps
-  // No key required.
   // eslint-disable-next-line no-unused-vars
-  const basemapOsmDefault = L.tileLayer(
-    'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+  const basemapOsmBw = L.tileLayer(
+    'https://{s}.tiles.wmflabs.org/bw-mapnik/{z}/{x}/{y}.png',
     {
       attribution:
-        'Map data: &copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+        '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>',
       minZoom: 0,
-      maxZoom: 17,
-      name: 'OSM - Default',
+      maxZoom: 18,
+      name: 'OSM - Mapnik B&W',
       type: 'png',
```

Comment:
I'm getting an error.  is their service down?
![Screenshot 2023-10-27 at 4 32 15 PM](https://github.com/TACC/protx-dashboard/assets/8287580/af03ac05-5f0d-48ca-b5e0-4291c53cad1f)